x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -879,6 +879,14 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         );
         expect(await testSubjects.isSelected('policyWindowsEvent_dns')).to.be(wasSelected);
       });
+
+      it('should show trusted apps card and link should go back to policy', async () => {
+        await testSubjects.existOrFail('fleedTrustedAppsCard');
```

Comment:
spelling: should it be `fleet*`?

```suggestion
        await testSubjects.existOrFail('fleetTrustedAppsCard');
```